app/routes/tunnels.py

```diff
@@ -49,20 +50,22 @@ def start_tunnel() -> Tuple[Response, int]:
         json_schema_manager.validate(request.json, "tunnel_create.json")
 
         subdomain_id = dig(request.json, "data/relationships/subdomain/data/id")
-        port_type = dig(request.json, "data/attributes/port")
+        port_types = dig(request.json, "data/attributes/port")
         ssh_key = dig(request.json, "data/attributes/sshKey")
 
         current_user = User.query.filter_by(uuid=get_jwt_identity()).first_or_404()
         try:
             tunnel_info = TunnelCreationService(
-                current_user, subdomain_id, port_type, ssh_key
+                current_user, subdomain_id, port_types, ssh_key
             ).create()
         except SubdomainLimitReached:
             return json_api(SubdomainLimitReached, ErrorSchema), 403
         except TunnelLimitReached:
             return json_api(TunnelLimitReached, ErrorSchema), 403
         except TunnelError:
             return json_api(TunnelError, ErrorSchema), 500
+        except RedisError:
+            return json_api(RedisError, ErrorSchema), 500
```

Comment:
We don't need to share this as a specific error I don't think unless there's some actionable thing the user can do.  It's fine to return a generic TunnelError